valid_parentheses.py

```diff
@@ -0,0 +1,38 @@
+""" Given a string s containing just the characters
```

Comment:
nice, little nit, you can new line before and after 

"""
Given a string s containing just the characters
...
Open brackets must be closed in the correct order.
"""